bin/cml/publish.js

```diff
@@ -11,6 +11,8 @@ exports.command = data ? 'publish' : 'publish <asset>';
 exports.description = 'Upload an image to build a report';
 
 exports.handler = async (opts) => {
+  opts.native = opts.native || opts.gitlabUploads;
```

Comment:
```suggestion
opts.native ||= opts.gitlabUploads;
```

Supporting Node.js 14 is dirving me crazy.

Reply:
but why do we have to support v14? Isn't `package.json::engines.node >= 16` enough?

Reply:
Refresher: https://github.com/iterative/cml/pull/966 & [postmortem](https://www.notion.so/iterative/2022-04-15-CML-6113d0f195b7430b81a601d508c12b1e); we _do_ have to support Node.js 14 for 10 days more, when it becomes end–of–life.

Reply:
active support [ended 6 months ago](https://endoflife.date/nodejs)?

Reply:
Update: I meant Node.js 12 all this time. 🙈

Reply:
but `maintenance` != `active`

Reply:
Yes, but some people still use `maintenance` versions. For example, does DVC support `maintenance` Python versions?

Reply:
`maintenance` means "we will fix security bugs & bump patch version", not "we will add new features & bump minor/major version"